.buildkite/pipeline.yml

```diff
@@ -7,28 +7,33 @@ env:
   CACHE_DIR: "/cache/cardano-ledger-specs"
 
 steps:
-  - label: Check Hydra evaluation of release.nix
-    commands:
-      - "nix-build -A iohkLib.check-hydra -o check-hydra.sh"
-      - "./check-hydra.sh"
-    agents:
-      system: x86_64-linux
+  #- label: Check Hydra evaluation of release.nix
+  #  commands:
+  #    - "nix-build -A iohkLib.check-hydra -o check-hydra.sh"
+  #    - "./check-hydra.sh"
+  #  agents:
+  #    system: x86_64-linux
```

Comment:
What's the intention behind commenting these out? If they're now redundant, let's remove them. Otherwise this suggests that this PR isn't ready for review yet

Reply:
There's eval errors that exist still with haskell.nix. They're benign, but we can't get a positive result. We've made the tools team aware of them and they are being worked on, but at this point, we were tasked with getting things updated and this is a risk we're willing to take.

The only thing that can happen here that would be bad is if someone breaks the eval to the point hydra can't generate it at all, you won't get a notification of a hydra failure and you have to go straight to hydra to see the errors. I agree, lets just remove the commented code, no reason to leave the comments in.

Also, this check has been notoriously unreliable and fails more than 50% of the time because it wants to run something on a darwin builder to complete the eval. I have a WIP approach being tested on node that we'll be adding to iohk-nix here https://github.com/input-output-hk/cardano-node/pull/555.